Makefile

```
@@ -272,33 +265,37 @@ get-token-and-cert:  ## Get a token from a running K8s cluster for local develop
 
 NAMESPACE ?= ""
 .PHONY: bundle
-bundle: manifests kustomize ## Generate bundle manifests and metadata, then validate generated files.
+bundle: operator-sdk manifests kustomize ## Generate bundle manifests and metadata, then validate generated files.
 	mkdir -p koku-metrics-operator/$(VERSION)/
 	rm -rf ./bundle koku-metrics-operator/$(VERSION)/
-	operator-sdk generate kustomize manifests
+	$(OPERATOR_SDK) generate kustomize manifests
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMAGE_SHA}
-	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
-	operator-sdk bundle validate ./bundle
+	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
 	cp -r ./bundle/ koku-metrics-operator/$(VERSION)/
 	cp bundle.Dockerfile koku-metrics-operator/$(VERSION)/Dockerfile
 	scripts/txt_replace.py $(VERSION) $(PREVIOUS_VERSION) ${IMAGE_SHA} --namespace=${NAMESPACE}
```

This fails unless the requirements are installed. Either mention the creation of a virtual environment and installation of the requirements in the prerequisites section, or take care of creating the environment and installing the requirements.

Something like this will work so long as a correct version of Python is available:
```suggestion
	.venv/bin/python scripts/txt_replace.py $(VERSION) $(PREVIOUS_VERSION) ${IMAGE_SHA} --namespace=${NAMESPACE}
```
```make
.PHONY: venv
venv:
	@python -m venv .venv
	@.venv/bin/python -m pip install -U pip
	@.venv/bin/python -m pip install -r scripts/requirements.txt
```